pages/common/rsync.md

```diff
@@ -11,6 +11,10 @@
 
 `rsync {{remote_host}}:{{path/to/remote_file}} {{path/to/local_directory}}`
 
+- Worry-free file transfer (handles soft-links, directories, access rights, and previously transferred files):
+
+`rsync -Laura {{from}} {{to}}`
```

Comment:
`{{from}}` and `{{too}}` should be made to match other examples, maybe as `{{/from/file}}` and `{{/to/file}}`, but I'm not sure.

Reply:
ok.